types/compare.go

```diff
@@ -35,6 +37,62 @@ func CompareUint64(x, y uint64) int {
 	return 1
 }
 
+//VecComparUU returns []int64 comparing the []uint64 x to []uint64 y
+func VecComparUU(n int, x, y []uint64, res []int64) {
```

Comment:
1. s/ Compar/ Compare
2. Add a test for this function.
3. so as the other 3 functions.

Reply:
I had modified Compar to Compare

Reply:
I had added some testcases for the 4 functions.